### ISSUE TITLE
Quickfix not working

### ISSUE THREAD

**Issue by xkeshav** [DavidAnson/vscode-markdownlint#314]
- vs code v 1.85 
- markdownlint **v 0.53.0** 
- macos v 14.12.0

vscode --> About
```txt
Version: 1.85.1 (Universal)
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:48:06.308Z (2 wks ago)
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0

```
I was using `**` for bold and then got the warning of MD036

```sh
MD036/no-emphasis-as-heading: Emphasis used instead of a headingmarkdownlint[MD036](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md036.md)
```
<img width="461" alt="Screenshot 2023-12-28 at 8 55 21 AM" src="https://github.com/DavidAnson/vscode-markdownlint/assets/415219/e5aa3ea3-8dfc-453e-a577-3023b7e7db38">

and when I click on quick fix and select  _fix all  supported markdownlint violations in thus document_ option
but nothing happens at all.

so my question is do I need to configure any settings ( workspace/user) to make this working?

MD036/no-emphasis-as-heading: Emphasis used instead of a headingmarkdownlint[MD036](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md036.md)

I have added .vscode/markdownlint.json file as per suggestions but still no luck.

someone can help and check the issue

**Comment by DavidAnson** [comment]
MD036 is not a fixable rule. That's why there is no menu item to fix the current instance in the image above. For violations of this rule, it's not possible to tell if the emphasized text was meant to be a heading and, if so, what heading level was intended.

You can see the list of fixable rules here: <https://github.com/DavidAnson/vscode-markdownlint#rules>.

**Comment by DavidAnson** [comment]
PS - I considered using the text "Fix all fixable violations..." which may be bore explicit, but it seemed too goofy and I went with "supported" instead.
